src/styles/main.scss

```diff
@@ -0,0 +1,111 @@
+main {
+  grid-template-columns: repeat(4, 200px);
+  gap: 46px 48px;
+  width: 944px;
+  padding: 50px 40px;
+  transition-duration: 0.3s;
+  display: grid;
+  margin: 0 auto;
+}
+
+.card {
+  border: 1px solid var(--gray-elements);
+  border-radius: 5px;
+  width: 198px;
+  height: 406px;
+  background-color: var(--white);
+  justify-content: center;
+
+  &:hover {
+    transform: scale(1.5);
+  }
```

Comment:
Fix it 
<img width="634" alt="image" src="https://github.com/user-attachments/assets/28cf68ff-41ab-44a4-bb01-2dc5453d98ea">